lib/json_info_codec.dart

```diff
@@ -330,16 +330,11 @@ class AllInfoToJsonConverter extends Converter<AllInfo, Map>
     };
   }
 
-  Map _visitDependencyInfo(DependencyInfo info) =>
-      {'id': info.target.serializedId, 'mask': info.mask};
-
   Map _visitAllInfoHolding(AllInfo allInfo) {
     var map = new SplayTreeMap<String, List>(compareNatural);
     void helper(CodeInfo info) {
       if (info.uses.isEmpty) return;
-      map[info.serializedId] = info.uses
-          .map((u) => _visitDependencyInfo(u))
-          .toList()..sort((Map a, Map b) => compareNatural(a['id'], b['id']));
+      map[info.serializedId] = info.uses.toList();
```

Comment:
Done.

> also we aren't sorting anymore?

We are, but it was only based on `id`. I was seeing shifting output because items with the same `id` and different masks were moving around.